packages/std/src/mock.rs

```diff
@@ -294,6 +298,7 @@ impl<C: DeserializeOwned> MockQuerier<C> {
     pub fn new(balances: &[(&str, &[Coin])]) -> Self {
         MockQuerier {
             bank: BankQuerier::new(balances),
+            #[cfg(feature = "staking")]
```

Comment:
Wow, I thought this got clippy/compile errors. I guess things got better since rust 1.45